osgar/drivers/rosmsg.py

```diff
@@ -548,7 +550,19 @@ def slot_raw(self, timestamp, data):
             s = data[6:].split(b' ')
             addr = s[0]
             msg = b' '.join(s[1:])
-            self.bus.publish('radio', [addr, msg])
+            if addr == b'base_station':
+                artifact_score = ign_pb2.ArtifactScore()
+                artifact_score.ParseFromString(msg)
```

Comment:
can this fail? And if yes what would be the content?

Reply:
yes, it can fail - I have it patched somehow in my wip copy